### PR TITLE
Bug 1159959 - Sticky submit button. r=dietrich

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: lib/views/create-bug.js [hunk 1]
```diff
@@ -220,8 +211,8 @@ function submitBug() {
     function createAttachments() {
       if (!bug.attachments.length) {
         localforage.setItem(KEY, null).then(function() {
-          dialog.close();
           app.page('/bug/' + result.id);
```
Comment (autonome) [inline lib/views/create-bug.js]: Is this change related to this bug?

Comment (daleharvey) [inline lib/views/create-bug.js]: Sorry no, was a small issue I noticed while writing the patch, it doesnt actually make any difference
